app/src/main/java/com/work/restaurant/data/source/local/road/RoadLocalDataSourceImpl.kt

```diff
@@ -82,17 +84,16 @@ class RoadLocalDataSourceImpl : RoadLocalDataSource {
 
             if (addressDatabase?.isOpen!!) {
                 if (getAddressCount != null) {
-                    callback.onSuccess(true)
+                    AppExecutors().mainThread.execute {
```

Comment:
```suggestion
                    appExecutors.mainThread.execute {
```
여기서 생성자를 매번 생성하는것 보다는 RoadLocalDataSourceImpl 의 생성자에서 주입받아서 사용하는것이 좋지 않을까요?

Reply:
56dda00 수정하였습니다.